numpy/core/src/multiarray/compiled_base.c

```diff
@@ -1401,7 +1416,15 @@ arr_add_docstring(PyObject *NPY_UNUSED(dummy), PyObject *args)
         Py_RETURN_NONE;
     }
 
-    if (!PyArg_ParseTuple(args, "OO!:add_docstring", &obj, &PyUnicode_Type, &str)) {
+    NPY_PREPARE_ARGPARSER;
+    if (npy_parse_arguments("add_docstring", args, len_args, NULL,
+            "", NULL, &obj,
+            "", NULL, &str,
+            NULL, NULL, NULL) < 0) {
+        return NULL;
+    }
+    if (!PyUnicode_Check(str)) {
+        PyErr_SetString(PyExc_RuntimeError, "argument docstring of add_docstring should be a str");
```

Comment:
Can you wrap the long line?
```suggestion
        PyErr_SetString(PyExc_RuntimeError,
                "argument docstring of add_docstring should be a str");
```
would work, or some other way (since no auto-style likes this way).